keyboards/work_louder/work_board/rules.mk

```diff
@@ -14,7 +14,7 @@ CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
```

Comment:
The board supports both rgb light and rgb matrix.

Reply:
Reverted, now it will throw `"Cannot use RGBLIGHT and RGB Matrix using WS2812 at the same time."` and the firmware will be too large.